### PR TITLE
ANT v2

### PR DESCRIPTION
**Pull request by Destiner**

### CODE REVIEW

File: data/metadataOverwrite.json [hunk 1]
```diff
@@ -25,6 +25,11 @@
 			"decimals": 9,
 			"symbol": "DGD",
 			"name": "DigixDAO DGD Token"
+		},
+		"0x960b236A07cf122663c4303350609A66A7B288C0": {
+			"decimals": 18,
+			"symbol": "ANT (old)",
```
Comment (bonustrack) [inline data/metadataOverwrite.json:31]: If this appear on the UI i would suggest something like: 
```suggestion
			"symbol": "ANT v1",
```

**Review by bonustrack — APPROVED** [review]

File: data/metadataOverwrite.json (hunk 1 above)
Comment (Destiner) [reply data/metadataOverwrite.json:31]: Works for me, but they suggested "(old)" suffix.

Comment (bonustrack) [reply data/metadataOverwrite.json:31]: Ok then we can keep old

Comment (sohkai) [reply data/metadataOverwrite.json:31]: Hey guys :). Yes, for token symbols we're trying to align with "ANT (old)" (ANTv1) and "ANT" (ANTv2) to make it less confusing for users, especially as v1 will disappear over time.

**Pull request merged**